src/__main__.py

```diff
@@ -36,11 +36,11 @@ def directory(dirname):
 # Command parameters (specify what the algorithm does):
 pcsf_params = parser.add_argument_group("PCSF Parameters")
 
-pcsf_params.add_argument("-w", dest="w", nargs="*", type=float, required=False, default=[6],
+pcsf_params.add_argument("-w", dest="w", nargs="*", type=float, required=False, default=6,
```

Comment:
Giving lists was causing an error since line 269 in graph.py would make dummy_costs a list of lists. I'm not sure this is the right solution, though - it seems like you did want them to be lists? Maybe for param screens?